library/src/webapp/js/headscripts.js

```diff
@@ -782,6 +782,9 @@ function includeWebjarLibrary(library) {
     document.write('\x3Cscript src="' + webjars + 'jquery.tablesorter/' + libraryVersion + '/dist/js/extras/jquery.metadata.min.js' + ver + '">' + '\x3C/script>');
     document.write('\x3Clink rel="stylesheet" href="' + webjars + 'jquery.tablesorter/' + libraryVersion + '/dist/css/theme.jui.min.css' + ver + '"/>');
     document.write('\x3Clink rel="stylesheet" href="' + webjars + 'jquery.tablesorter/' + libraryVersion + '/dist/css/jquery.tablesorter.pager.min.css' + ver + '"/>');
+	} else if (library === "jquery.asmselect") {
+		libraryVersion = "1.0.4a-beta";
+		document.write('\x3Clink rel="stylesheet" href="/library/js/jquery/asmselect/' + libraryVersion + '/jquery.asmselect.css' + ver + '"/>');
```

Comment:
this doesn't seem right to me but I don't know how to make the file appear under the webjars directory vs webapps/library/js during the deploy process.

Do these libraries have to be in webjars.org? I tried adding
```xml
<!--Used in Delegated Access, Site Manage -->
        <dependency>
            <groupId>org.webjars</groupId>
            <artifactId>asmselect</artifactId>
            <version>1.0.4a-beta</version>
            <scope>runtime</scope>
        </dependency>
```
to the POM but that created a build error and I couldn't find asmselect on webjars.org.